File: Language_Coder/src/loop/Code548.java
package loop;

public class Code548 {

	public static void main(String[] args) {
		for (int i = 2; i <= 4; i++) {
			for (int j = 1; j <= 5; j++) {
				if (j != 1)
					System.out.print("   ");
				if(i * j < 10)
					System.out.print(i + " * " + j + " =  " + i * j);
				else
					System.out.print(i + " * " + j + " = " + i * j);
			}
			System.out.println();
		}
	}

}

File: Language_Coder_Cpp/Code548/Code548.cpp
// Code548.cpp

#include <iostream>
using namespace std;

int main()
{
	for (int i = 2; i <= 4; i++) {
		for (int j = 1; j <= 5; j++) {
			cout << i << " * " << j << " =";

			if (i * j >= 10) {
				cout << " ";
			}
			else {
				cout << "  ";
			}

			cout << i * j<<"   ";
		}
		cout << endl;
	}
}